### DISCUSSION TITLE
Why doesn't Mac use the server podman and podman-remote like in Linux?

### DISCUSSION THREAD

**Discussion by YevheniiPokhvalii** [containers/podman#19429] [General]
When you install `podman` with brew on Linux, it will install both `podman` and `podman-remote` - server and client.
```
$ which podman
/home/linuxbrew/.linuxbrew/bin/podman
$ which podman-remote
/home/linuxbrew/.linuxbrew/bin/podman-remote
```
`podman` (server) is ready and easy to use out of the box in Linux and I don't need to spin up any additional machines.
Why does Mac only have `podman` with the `podman-remote` functionality? which requires:
```
podman machine init
podman machine start
```
Why does it not work the same way like it does in Linux?

**Comment by afbjorklund** [comment]
The `podman` program is Linux-only. On Mac and Windows, `podman-remote` is renamed to just `podman`.

**Reply by YevheniiPokhvalii** [reply]
I see. Could you explain what's so different about Mac? Is it something about virtualization? 
I thought BSD system is somewhat similar to Linux.

**Reply by rhatdan** [reply]
Containers are a linux concept they do not exist in BSD worlds.  Podman will work with FreeBSD and Jails, as far as I know, but no one has tried to make this work on a Mac, or even if it is possible.

**Reply by afbjorklund** [reply]
It is possible to run Podman Machine on Linux too, if you want to run the containers on a second kernel (in a VM)

Or some alternative runtimes, that run each container in a new virtual machine. Including the new ones, for WebAssembly

**Comment by dfr** [comment]
I believe that it might be possible to make something vaguely
container-like using macos sandboxes but I don't know for sure and I'm not
going to try to find out. Getting podman and buildah working well on
FreeBSD is enough to keep me busy.

Doug.

**Reply by YevheniiPokhvalii** [reply]
Thank you. Now I understand. I found this thread that seems to be related to the subject: 
https://macoscontainers.org/
https://github.com/containerd/containerd/discussions/5525
